### ISSUE TITLE
Bot to add post from rss feed

### ISSUE THREAD

**Issue by maclandrol** [legaultmarc/genomenews#5]
Write a bot to add post from popular journals rss feed (nature, plos, bmc, etc)

__Current idea__
Bots should be distinguishable from regulars Users. Maybe add a boolean field in UserProfile and set the default value to 0. This field should not be shown in in userprofile detail or any other template

**Comment by legaultmarc** [comment]
I agree with what you say here. I also think that having twitter bots could be interesting. For bots, it would also be cool to include them in the admin interface so we can do some manual curation to avoid irrelevant posts.
